data/taxRates.ts

```diff
@@ -0,0 +1,55 @@
+const taxRates = {
+  AK: 0.0,
+  AL: 0.04,
+  AR: 0.065,
+  AZ: 0.056,
+  CA: 0.0725,
+  CO: 0.029,
+  CT: 0.0635,
+  DC: 0.06,
+  DE: 0.0,
+  FL: 0.06,
+  GA: 0.04,
+  HI: 0.04,
+  IA: 0.06,
+  ID: 0.06,
+  IL: 0.0625,
+  IN: 0.07,
+  KS: 0.065,
+  KY: 0.06,
+  LA: 0.0445,
+  MA: 0.0625,
+  MD: 0.06,
+  ME: 0.055,
+  MI: 0.06,
+  MN: 0.06875,
+  MO: 0.04225,
+  MS: 0.07,
+  MT: 0.0,
+  NC: 0.0475,
+  ND: 0.05,
+  NE: 0.055,
+  NH: 0.0,
+  NJ: 0.06625,
+  NM: 0.05125,
+  NV: 0.0685,
+  NY: 0.04,
+  OH: 0.0575,
+  OK: 0.045,
+  OR: 0.0,
+  PA: 0.06,
+  RI: 0.07,
+  SC: 0.06,
+  SD: 0.045,
+  TN: 0.07,
+  TX: 0.0625,
+  UT: 0.061,
+  VA: 0.53,
+  VT: 0.06,
+  WA: 0.065,
+  WI: 0.05,
+  WV: 0.06,
+  WY: 0.04,
+};
+
+export default taxRates;
```

Comment:
Good work. I would name the variable StateTaxRate.

Reply:
Noted!